src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -495,10 +663,12 @@ public void processTasks(List<Row> wbs, List<Row> tasks, List<Row> costs, List<R
          {
             task = parentTask.addTask();
          }
+         task.setProject(m_project.getProjectProperties().getName()); // P6 task always belongs to project
```

Comment:
Not sure this is worth populating, unless you actually need it? It feels like a very "MS Project" thing?